packages/dds/sequence/src/intervalTree.ts

```diff
@@ -20,19 +20,55 @@ export interface AugmentedIntervalNode {
 
 export const integerRangeToString = (range: IIntegerRange) => `[${range.start},${range.end})`;
 
+/**
+ * Basic interval abstraction
+ */
 export interface IInterval {
+    /**
+     * @returns a new interval object with identical semantics.
+     */
     clone(): IInterval;
+    /**
+     * Compares this interval to `b` with standard comparator semantics:
+     * - returns -1 if this is less than `b`
+     * - returns 1 if this is greater than `b`
+     * - returns 0 if this is equivalent to `b`
+     * @param b - Interval to compare against
+     */
     compare(b: IInterval): number;
+    /**
+     * Compares the start endpoint of this interval to `b`'s start endpoint.
+     * Standard comparator semantics apply.
+     * @param b - Interval to compare against
+     */
     compareStart(b: IInterval): number;
+    /**
+     * Compares the end endpoint of this interval to `b`'s end endpoint.
+     * @param b - Interval to compare against
```

Comment:
```suggestion
     * Compares the end endpoint of this interval to `b`'s end endpoint.
     * Standard comparator semantics apply.
     * @param b - Interval to compare against
```
Nit: for consistency?